spec/7/controllers/main.rb

```diff
@@ -0,0 +1,8 @@
+
+class MainController < Kenji::Controller
+
+  get '/crasher' do
+    raise 'kaboom!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.